settings.gradle

```diff
@@ -1,2 +1,4 @@
 rootProject.name = 'runge-world-cities'
+include 'src:test:java'
+findProject(':src:test:java')?.name = 'java'
```

Comment:
This looks like something IntelliJ put in but it's wrong.

Reply:
im not sure where to locate this.